client/src/api/index.js

```diff
@@ -112,5 +148,48 @@ export default {
 
       return [tokenRequest, userRequest]
     }
+  },
+  teams: {
```

Comment:
This language is a bit strange. I think they're called "organizations" throughout the backend at least- are we calling them teams on the frontend?

Reply:
yeah i wanted the api to reflect what the front end is saying so it is easier to keep straight